README.md

```diff
@@ -134,6 +134,9 @@ API services currently supported are:
 * sentry_dsn
 * pocket_key
 
+##### ADM API key
+To use ADM in debug builds, there must be a `<project-dir>/app/src/main/assets/api_key.txt` file that contains the api key. It is necessary that it is in the project's assets folder. See [ADM integration doc][adm] for more info.
```

Comment:
Where can developers get an ADM API key? Is there a debug key we can use? I'd rather not have all of us using the production API keys during development.

Reply:
I added it to where we have the other keys. It is a debug key

Reply:
I don't think we would want to link to that folder so that's why I didn't mention where to get it. I'll send an email to the team about it.

Reply:
Gotcha, that sounds good.

To be clear, is this the production key or the debug key?

Reply:
Debug